tests/unit-tests/test_construct_graph.py

```diff
@@ -188,6 +188,47 @@ def test_feat_ops():
     assert "attention_mask" in proc_res
     assert "token_type_ids" in proc_res
 
+    # Compute BERT embeddings.
+    feat_op3 = [
```

Comment:
Do you add test for GPU?

Reply:
it's quite hard to test if the embeddings are computed by GPU.

Reply:
We would better test whether the code works with multi-GPU and generate same (similar) embedding as CPU.
We can add a new unitest pipeline for multi-GPU unitests.

Reply:
i just realized that the code is tested on GPU. if the machine has GPU available, it uses GPU directly. Multi-GPU setting cannot only be tested in the integration test. The code requires to generate multiple input files to trigger multi-GPU tests. this is tested in the integration test.

Reply:
OK